src/main/diagnostics/steps/step5.browserWindows.ts

```diff
@@ -17,11 +17,11 @@ const stepDescriptiveName = 'BrowserWindowsChecks';
 const run = async (logger: ElectronLog): Promise<DiagnosticStepResponse> => {
     try {
         /** Main window check */
-        if (!windowManager.mainWindowReady) {
+        if (!MainWindow.isReady) {
```

Comment:
is just logging the message that the main window is not ready enough at this point? don't we want to know why is not ready? I'm not sure at this point what is being diagnosed and what do we do with the results of this.

Reply:
I know it has to be visible for this to work, though I'm not sure in what case we're running diagnostics without the window visible. This stuff is a bit out of scope here (it was just updated) and I'm not super familiar with it so I'm going to leave it for now.